### PR TITLE
ADD: SignUp page and routes to pages

### PR DESCRIPTION
**Pull request by abbosjon99**

### CODE REVIEW

File: frontend/src/app/signin/page.tsx [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import SignIn from "@/src/components/auth/signin/signin";
+
+const SignInPage = () => {
+  return (
+    <main className="sign-background">
```
Comment (coronado03) [inline frontend/src/app/signin/page.tsx:5]: Remove div and put css in main

File: frontend/src/app/signup/page.tsx [hunk 2]
```diff
@@ -0,0 +1,13 @@
+import SignUp from "@/src/components/auth/signup/signup";
+
+const SignUpPage = () => {
+  return (
+    <main className="sign-background">
```
Comment (coronado03) [inline frontend/src/app/signup/page.tsx:5]: same here

File: frontend/src/components/auth/signin/signin.tsx [hunk 3]
```diff
@@ -19,7 +19,10 @@ const SignIn = () => {
         </div>
         <div className="text-center">
           <span>Don't Have An Account?</span>
-          <a href="/" className="text-foreground-primary inline-block pl-2">
+          <a
```
Comment (coronado03) [inline frontend/src/components/auth/signin/signin.tsx:22]: https://nextjs.org/docs/pages/api-reference/components/link

use next link instead for performance

File: frontend/src/components/auth/signup/signup.tsx [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+            inputName="Password"
+            placeHolder="Enter your pasword"
+            isPassword
+          />
+          <Input placeHolder="Repeat your pasword" isPassword />
+        </div>
+        <Button label="Create account" />
+        <div className="text-center">
+          <span>Already Have An Account?</span>
+          <a
```
Comment (coronado03) [inline frontend/src/components/auth/signup/signup.tsx:23]: Next Link here too

File: frontend/src/app/page.tsx [hunk 5]
```diff
@@ -1,10 +1,10 @@
-import SignIn from "../components/auth/signin/signin";
+import SignUp from "../components/auth/signup/signup";
 
 const LandingPage = () => {
   return (
-    <main className="bg-slate-400">
-      <div className="w-screen flex justify-center h-[500px] items-center gap-4">
-        <SignIn />
+    <main className="sign-background">
+      <div className="w-screen flex justify-center h-screen items-center gap-4">
```
Comment (coronado03) [inline frontend/src/app/page.tsx:6]: remove gap-4

File: frontend/src/app/signin/page.tsx [hunk 6]
```diff
@@ -0,0 +1,13 @@
...
+      <div className="w-screen flex justify-center h-screen items-center gap-4">
```
Comment (coronado03) [inline frontend/src/app/signin/page.tsx:6]: remove gap-4

**Review by coronado03 — CHANGES_REQUESTED** [review]
LGTM!  just a few comments

File: frontend/src/app/signin/page.tsx (hunk 1 above)
Comment (abbosjon99) [reply frontend/src/app/signin/page.tsx:5]: ✅

File: frontend/src/app/signup/page.tsx (hunk 2 above)
Comment (abbosjon99) [reply frontend/src/app/signup/page.tsx:5]: ✅

File: frontend/src/app/page.tsx (hunk 5 above)
Comment (abbosjon99) [reply frontend/src/app/page.tsx:6]: ✅

File: frontend/src/app/signin/page.tsx (hunk 6 above)
Comment (abbosjon99) [reply frontend/src/app/signin/page.tsx:6]: ✅

File: frontend/src/components/auth/signin/signin.tsx (hunk 3 above)
Comment (abbosjon99) [reply frontend/src/components/auth/signin/signin.tsx:22]: ✅

File: frontend/src/components/auth/signup/signup.tsx (hunk 4 above)
Comment (abbosjon99) [reply frontend/src/components/auth/signup/signup.tsx:23]: ✅

**Pull request merged**